dashboard/app/views/home/_homepage.html.haml

```diff
@@ -48,12 +48,17 @@
 
 .clear{style: "clear: both"}
 
+-# In-page reminders and surveys
 - if current_user
   - if current_user.teacher?
     #terms_reminder
       - unless current_user.accepted_latest_terms?
         = render template: 'api/accept_terms_reminder'
         = render partial: 'layouts/terms_interstitial'
+      - if show_nps_survey? SurveyResult::NET_PROMOTER_SCORE_2017
```

Comment:
I decided to move it here because the `index.html.haml` page contained the modal pop-ups like asking for school and accepting the terms of service. I thought this would be clearer that here are all the in-page reminders that show above the classroom sections.